Pully Penelope/Assets/Scripts/Blade.cs

```diff
@@ -4,11 +4,26 @@
 
 public class Blade : MonoBehaviour
```

Comment:
So this script is attached to the blade of the guillotine. It doesn't actually change whether the game is won or lost, but just observes if the game is won. If player wins the game, then I wanted to deactivate the blade so that they couldn't lose after they had already won.